brainstormforce/ultimate-addons-for-gutenberg
# Fix: Lottie block - Not working on frontend due SVG https.

### Description

- Removed added s for HTTP to resolve the issue.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Lottie animation and add any Lottie URL or JSON file in `next-release`.
- Update and check frontend. It will not visible on front end.
- Now switch to `lottie-not-working-on-frontend` branch and refresh the page It will be visible.

### Checklist:
